src/test/kotlin/lotto/domain/LottoPurchaseAmountTest.kt

```diff
@@ -0,0 +1,15 @@
+package lotto.domain
+
+import org.assertj.core.api.Assertions.assertThatThrownBy
+import org.junit.jupiter.api.Assertions.*
+import org.junit.jupiter.api.Test
+
+class LottoPurchaseAmountTest {
+
+    @Test
+    fun `throw exception when a value is not the multiples of 1,000`() {
+        assertThatThrownBy{ LottoPurchaseAmount(1001) }
+            .isInstanceOf(IllegalArgumentException::class.java)
+    }
```

Comment:
```suggestion
    @Test
    fun `throw exception when a value is not the multiples of 1,000`() {
        assertThrows<IllegalArgumentException> {
            LottoPurchaseAmount(1001)
        }
    }
```

which one do you prefer?

Reply:
I don't have any preference but I'm just used to assertj. For me, the latter one seems more readable. I could use junit assertion!